### PR TITLE
Feature/dc 1512 add linters

### PR DESCRIPTION
**Pull request by oleksii-symon-corva-ai**
### Rationale
We want to ensure code quality and consistency.

### Changes
Added:
* isort to format imports
* autoflake to remove usused imports, vars and `pass`es
* black to format the code

[JIRA ticket](https://corvaqa.atlassian.net/browse/DC-1512)

#### TODO
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -2,7 +2,9 @@
 testpaths = tests
 
 [flake8]
-max-line-length = 120
+select = C, E, W, F, I
+max-line-length = 88
+extend-ignore = E203, W503
```
Comment (ville) [inline setup.cfg:7]: Why E203 is ignored?

**Review by ville — APPROVED** [review]

File: setup.cfg (hunk 1 above)
Comment (oleksii-symon-corva-ai) [reply setup.cfg:7]: https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#flake8

**Review by jordanambra — APPROVED** [review]
Looks good. I assume the formatting changes are from actually running Black, right?

**Comment by oleksii-symon-corva-ai** [comment]
> I assume the formatting changes are from actually running Black, right?

Yes, Black was run on the whole project and fixed some files. Also I've set line length to be 88 chars at max, as this is what Black expects, so fixed those issues too.

**Pull request merged**